templates/SilverStripe/Forms/CheckboxField_holder.ss

```diff
@@ -1,7 +1,10 @@
-<div id="{$HolderID.ATT}" class="field form-group checkbox {$ExtraClass.ATT} <% if $Message %>has-error<% end_if %>">
```

Comment:
Is `checkbox` implicit given it's a CheckboxField?

Reply:
I've removed the `checkbox` because it's done in `form-check` holders now.